testing/tests/DevExpress.ui.widgets.scheduler/integration.workSpace.tests.js

```diff
@@ -1721,11 +1721,15 @@ module('Resource Cell Template', () => {
                         ],
                     }],
                     resourceCellTemplate: function(cellData, cellIndex, cellElement) {
-                        if(!cellIndex) {
-                            const $cell = $(cellElement);
-                            assert.equal(getOuterWidth($cell), 100, 'Resource cell width is OK');
-                            assert.roughEqual(getOuterHeight($cell), 276, 1.001, 'Resource cell height is OK');
-                        }
+                        const done = assert.async();
+                        setTimeout(() => {
```

Comment:
It's a comment to discuss.

Maybe, we can use a fake setTimeout in the old QUnit tests?
Something like:
https://www.npmjs.com/package/@sinonjs/fake-timers

But, I'm not sure that this package is suitable for us, because we have a very old version of Sinon 
(we use 1.10.3, the actual version is 14.0.0)

Reply:
This approach doesn't work for us because our templating engine needs to render asynchronously.